CMakeLists.txt

```diff
@@ -45,7 +45,11 @@ set(BUILD_SHARED_LIBS OFF)
 # CRT Libraries
 add_subdirectory(crt/aws-c-common)
 if (UNIX AND NOT APPLE)
-    set(DISABLE_PERL ON)
+    if (CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64 OR
+        CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l)
+    else()
```

Comment:
Could we collapse this into a single else-less check?

Reply:
Done.